pubsub/pubsub.go

```diff
@@ -168,8 +146,13 @@ type Message struct {
 }
 
 // Ack acknowledges the message, telling the server that it does not need to be
-// sent again to the associated Subscription. It returns immediately, but the
-// actual ack is sent in the background, and is not guaranteed to succeed.
+// sent again to the associated Subscription. It will be a no-op for some
+// providers; see
+// https://godoc.org/gocloud.dev/pubsub#hdr-At_most_once_and_At_least_once_Delivery
+// for more info.
+//
+// Ack returns immediately, but the actual ack is sent in the background, and
+// s not guaranteed to succeed.
```

Comment:
s -> is

Also mention that if ack fails, that will eventually show up in the error from Receive.

Reply:
Done.